1.5.0

```diff
@@ -0,0 +1,42 @@
+{
+    "instructionList": [
+        {
+            "sObjectName": "Account",
+            "instructionTypeName": "command__AC Account (Household) Layout",
+            "taskList": [
+                {
+                    "jiraTicket": "ac-23908a",
+                    "description": "Add Stage field below AccountName in household Layout",
+                    "actionType": "Field",
+                    "actionTypeName": "command__Account_Stage__c",
+                    "optionsMap": {
+                        "doNotModifyExisting": "true"
+                    },
+                    "location": {
+                        "position": "Below",
+                        "reference": "Name",
+                        "section": "Account Information",
+                        "column": 1,
```

Comment:
I think this is supposed to go in column 0. The page layout shows the account name in col 0. Even though it places it correctly as is.